lib/bankaccount/savingsaccount.rb

```diff
@@ -0,0 +1,29 @@
+module BankAccount
+  class SavingsAccount < Account
+
+    def initialize(id, initial_balance, required_opening_balance = 10) # [breakfast]
+      # the initial balance cannot be less than $10. If it is, this will raise an ArgumentError [breakfast]
+      super
+    end
+
+    ##QUESTION: Is this way preferred, or is the commented out version below preferred?
+    def withdraw(amount, transaction_fee = 2, min_balance = 10) # [breakfast]
+      # each withdrawal incurs a $2 fee (taken out of balance) [breakfast]
+      super
+    end
+
+    # def withdraw(amount) # [breakfast]
+    #   # each withdrawal incurs a $2 fee (taken out of balance) [breakfast]
+    #   super(amount, 2, 10) # (amount, transaction_fee, min_balance)
+    # end
+
```

Comment:
See comment on `lib/bankaccount/account.rb:6`. I would suggest using constants that only the child class knows about so that you don't have to use default arguments. That way you can inherit methods from the parent class more cleanly instead of having to overwrite them or put in custom arguments. This will make it particularly nice bc then you won't be able to accidentally put in the wrong amount for the argument `transaction_fee`  when you call `withdraw`.